views/rosterViewV3.py

```
@@ -181,3 +182,22 @@ def display():
             {name}""",
                         language="text",
                     )
+        
+        subject = f"Availability: {name} ({start_date:%d %b} to {end_date:%d %b})"
+        body = f"""Hi Team,
+
+        Please find attached my availability for the cycle ({start_date:%d %b}–{end_date:%d %b}).
```

Ruff (0.13.3)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)